Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,11 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+/*
+1. Math.random(): it is a function that generates a random decimal number between 0 and 1.
```

Comment:
"Between 0 and 1" is not specific enough (especially in program specification) because it does not clearly specify whether 0 and 1 are included or not.

Can you clarify whether the return value of `Math.random()` includes 0 and/or 1?